docs/snippets/common/component-styled-variables-template-literals.js.mdx

```diff
@@ -0,0 +1,6 @@
+```js
+const Component = styled.div`
+  background: `${props => props.theme.background.app}`
+  width: 0;
+`;
```

Comment:
I think styled components is react-specific too right?

Reply:
[Vue supports it as well](https://github.com/styled-components/vue-styled-components). I left the snippet there, so that when we have a proper react and vue example we can show them both. But i'll adjust accordingly

Reply:
OK